ote/src/cljs/ote/theme/colors.cljs

```diff
@@ -1,5 +1,21 @@
 (ns ote.theme.colors)
 
+(def dark-blue "#0034ac")
+(def dark-green "#005f61")
+(def basic-purple "#73468c")
+(def dark-purple "#520076")
+
+(def dark-red "#b40000")
+(def dark-gray "#505064")
+(def basic-brown "#784b28")
```

Comment:
This is just a note, no changes needed - once the whole palette is here, I'd reorder these into some logical order, eg. all greens in order basic-dark-light, or first basic colors, then dark colors, then light colors. Now they're in a bit of mix&match order.

Reply:
Currently they are in order given in the brand book, then black and white, which actually could be first before these accessible colors. But good idea and I'll make the order more logical in the near future.